src/js/index.js

```diff
@@ -125,10 +125,23 @@ RequestUtil.json = function(options = {}) {
     );
   }
 
-  // Plugins events
-  PluginSDK.Hooks.addAction("pluginsConfigured", onPluginsLoaded);
-  ConfigStore.addChangeListener(CONFIG_ERROR, onConfigurationError);
+  function startApplication() {
+    // Plugins events
+    PluginSDK.Hooks.addAction("pluginsConfigured", onPluginsLoaded);
+    ConfigStore.addChangeListener(CONFIG_ERROR, onConfigurationError);
 
-  // Load configuration
-  ConfigStore.fetchConfig();
+    // Load configuration
+    ConfigStore.fetchConfig();
+  }
+
+  if (!global.Intl) {
+    require.ensure(["intl", "intl/locale-data/jsonp/en.js"], function(require) {
```

Comment:
Good point. It might even makes sense to create a list of needed locals and use that to require the respective changes.